### ISSUE TITLE
Add remap host request

### ISSUE THREAD

**Issue by claytonneal** [detro/browsermob-proxy-client#3]
Please add the following request to your client!

POST /proxy/[port]/hosts - Overrides normal DNS lookups and remaps the given hosts with the associated IP address

**Comment by detro** [comment]
Oh wow! I have a user? :)

**Comment by claytonneal** [comment]
Yes you do! Great time saver for my implementation of browsermob
